scripts/run_liminal_runtime_benchmarks.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export NODE_ID=5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH
+
+cargo run --release -p aleph-node --features liminal-runtime-benchmarks -- bootstrap-chain \
+    --base-path /tmp/ \
+    --account-ids $NODE_ID \
+    --sudo-account-id $NODE_ID \
+    --chain-id a0snarknet \
+    --token-symbol SNRKZERO \
+    --chain-name 'Aleph Zero Snarknet' \
+    > ./chainspec.json
```

Comment:
Doesn't this need a `&` at the end to actually proceed to the next command?

Reply:
no, why? generating chainspec is an immediate action

Reply:
works for me at least xD